crates/bench/src/spacetime_raw.rs

```diff
@@ -74,7 +74,11 @@ impl BenchDatabase for SpacetimeRaw {
 
     fn count_table(&mut self, table_id: &Self::TableId) -> ResultBench<u32> {
         self.db.with_auto_commit(&ExecutionContext::default(), |tx| {
-            Ok(self.db.iter(tx, *table_id)?.map(|_| 1u32).sum())
+            Ok(self
```

Comment:
Is it worth avoiding calling `ExecutionContext::default()` more than once? I guess I don't care very much in benchmarking/testing/... code.